src/clinical/donor-repo.ts

```diff
@@ -217,7 +217,7 @@ export const donorDao: DonorRepository = {
     // Specific Requests for Donor documents can be paginated at the MongoDB level
 
     const limit =
-      entityTypes.includes('donor') && pageSize
+      entityTypes === ['donor'] && pageSize
```

Comment:
This will be comparing by "reference". I took a look and don't understand the logic here.

Is the goal to check if `entityTypes` _only_ has `donor`?

Reply:
You're right we are checking if entityTypes only includes donor
There's likely a better way that isn't checking by reference
Basically we are only concerned with pageSize at this stage if we are querying Donors specifically
All other entities we need to query all results, else we don't know the total number of records (b/c entities are stored as properties of Donors)

Reply:
Gotcha. Sorry I should clarify. 
``` javascript
['donor'] === ['donor']; // this will always be false because they're different objects
```
To check if entityTypes only includes donor, something like this should work:
``` javascript
entityTypes.length === 1 && entityTypes.includes('donor');
```

Reply:
Updated